File: Python/O(N)/on_71.py
# Source: https://www.geeksforgeeks.org/program-for-mean-and-median-of-an-unsorted-array/

# Python3 program to find mean
 
# Function for calculating mean
 
 
def findMean(a, n):
 
    sum = 0
    for i in range(0, n):
        sum += a[i]
 
    return float(sum/n)
 
 
# Driver code
a = [1, 3, 4, 2, 7, 5, 8, 6]
n = len(a)
 
# Function call
print("Mean =", findMean(a, n))

File: Java/O(N)/ON_71.java
// Source: https://www.geeksforgeeks.org/c-program-find-largest-element-array/

// Java program to
// find maximum in
// arr[] of size n
import java .io.*;
import java.util.*;
 
class ON_71
{
     
    // returns maximum in
    // arr[] of size n
    static int largest(int []arr,  
                       int n)
    {
        Arrays.sort(arr);
        return arr[n - 1];
    }
 
    // Driver code
    static public void main (String[] args)
    {
        int []arr = {10, 324, 45,
                     90, 9808};
        int n = arr.length;
        System.out.println(largest(arr, n));
    }
}
